test/unit/testDeceasedDod.js

```diff
@@ -14,27 +14,49 @@ describe('DeceasedDod', () => {
         });
     });
 
-    describe('handlePost()', () => {
-        let ctx;
-        let errors;
-        let formdata;
-        let session;
-        let hostname;
-        let featureToggles;
+    describe('getContextData()', () => {
+        it('should return the ctx with the deceased address and the screening_question feature toggle on', (done) => {
+            const req = {
+                sessionID: 'dummy_sessionId',
+                session: {form: {}, featureToggles: {screening_questions: true}},
+                body: {
+                    dod_day: '02',
+                    dod_month: '03',
+                    dod_year: '1952'
+                }
+            };
+            const ctx = DeceasedDod.getContextData(req);
+            expect(ctx).to.deep.equal({
+                dod_day: 2,
+                dod_month: 3,
+                dod_year: 1952,
+                dod_date: '1952-03-02T00:00:00.000Z',
+                dod_formattedDate: '2 March 1952',
+                isToggleEnabled: true,
+                sessionID: 'dummy_sessionId'
+            });
+            done();
+        });
 
-        it('should return the ctx with the deceased dod and the screening_question feature toggle', (done) => {
-            ctx = {
-                dob_day: '02',
-                dob_month: '03',
-                dob_year: '1952'
+        it('should return the ctx with the deceased address and the screening_question feature toggle off', (done) => {
```

Comment:
Please update the description for this test as it is for deceased dod but it is referring to deceased address.